assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -38,19 +43,34 @@ export const actions = {
 	 */
 	submitChanges( slug ) {
 		invariant( slug, 'slug is required.' );
-		return ( function* () {
-			const registry = yield Data.commonActions.getRegistry();
-
-			const { submitChanges } = registry.dispatch( `modules/${ slug }` ) || {};
-			if ( !! submitChanges ) {
-				return submitChanges();
-			}
 
-			return { error: `'modules/${ slug }' does not have a submitChanges() action.` };
+		return ( function* () {
+			return yield {
+				payload: { slug },
+				type: SETTINGS_SUBMIT_CHANGES,
+			};
 		}() );
 	},
 };
 
+export const controls = {
+	[ SETTINGS_SUBMIT_CHANGES ]: createRegistryControl( ( registry ) => async ( { payload } ) => {
+		const { slug } = payload;
+		const storeName = registry.select( CORE_MODULES )?.getModuleStoreName( slug );
+
+		if ( ! storeName ) {
+			return { error: `The module ${ slug } does not have a store.` };
```

Comment:
Similar to how it was before with the store name, let's quote the module slug in the string.
```suggestion
			return { error: `The module '${ slug }' does not have a store.` };
```

Reply:
Updated.